### PR TITLE
feat: description tooltip

### PR DESCRIPTION
**Pull request by load1n9**

### CODE REVIEW

File: www/routes/index.tsx [hunk 1]
```diff
@@ -49,7 +49,12 @@ export default function Home(props: PageProps<Example[]>) {
         <ul class="mt-6 text-gray-900">
           {props.data.map((example) => (
             <li>
-              <a href={`/${example.id}`} class="underline">
+              <a
+                href={`/${example.id}`}
+                class="underline"
+                data-bs-toggle="tooltip"
```
Comment (kt3k) [inline www/routes/index.tsx:55]: This looks like bootstrap specific data attribute. Is this meaningful here? (Or am I missing something?)

**Review by kt3k — COMMENTED** [review]
Thanks for your PR. Left one question

File: www/routes/index.tsx (hunk 1 above)
Comment (load1n9) [reply www/routes/index.tsx:55]: pretty sure it's how you do tooltips in tailwind

Comment (kt3k) [reply www/routes/index.tsx:55]: I found some usage of `data-bs-toggle` with tailwind such as [Windstrap](https://windstrap.netlify.app/#tooltips) or [Tailwind Elements](https://tailwind-elements.com/docs/standard/components/tooltip/), but we don't use either of them

Comment (load1n9) [reply www/routes/index.tsx:55]: I'll change this when I get back from vacation

**Review by kt3k — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**